module/metrics/transaction_validation.go

```diff
@@ -0,0 +1,45 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/module"
+)
+
+// TransactionValidationCollector TODO
+type TransactionValidationCollector struct {
+	transactionValidated        *prometheus.CounterVec
+	transactionValidationFailed *prometheus.CounterVec
+}
+
+// interface check
+var _ module.TransactionValidationMetrics = (*TransactionValidationCollector)(nil)
+
+func NewTransactionValidationCollector() *TransactionValidationCollector {
+	return &TransactionValidationCollector{
+		transactionValidated: promauto.NewCounterVec(prometheus.CounterOpts{
+			Name:      "transaction_validated_total",
+			Namespace: namespaceAccess,
+			Subsystem: subsystemTransactionValidation,
+			Help:      "counter for the validated transactions",
+		}, []string{"txID"}),
```

Comment:
we won't be able to use `txID` as a label for these. Since the value changes for every tx, this would create a new counter time series for every transaction, resulting in millions of new metrics per day. Labels can only be used when there's a small fixed set of possible values. `reason` below is good though.